src/components/Icon/Icon.jsx

```diff
@@ -133,14 +130,11 @@ Icon.propTypes = {
 };
 
 Icon.defaultProps = {
-  onClick: NOOP,
   className: "",
   icon: "",
-  clickable: true,
```

Comment:
@SergeyRoyt this is breaking change - please add this to the backlog and consider it when upgrading major